pcsx2/gui/Panels/PluginSelectorPanel.cpp

```diff
@@ -711,7 +711,7 @@ void Panels::PluginSelectorPanel::OnEnumComplete( wxCommandEvent& evt )
 
 				for( int i = 0; i < count; i++ )
 				{
-					auto str = m_ComponentBoxes->Get(pid).GetString(i);
+					auto str = m_ComponentBoxes->Get(pid).GetString(i).Lower();
```

Comment:
Maybe worth adding it to gsdx as well?

Reply:
I could, and lowercase the strings. It wasn't absolutely necessary, though, and I didn't feel like messing up the gsdx autodetection accidentally.